app/server-utils/stampy.ts

```diff
@@ -1,15 +1,21 @@
-import {withCache} from '~/server-utils/kv-cache'
-import {Converter} from 'showdown'
+import { withCache } from '~/server-utils/kv-cache'
+import { Converter } from 'showdown'
```

Comment:
this code style doesn't look like what we set up in `prettier` - can you enable format on save in your editor and/or  `npm run lint` from command line, please?